errai-ioc/src/main/java/org/jboss/errai/ioc/client/api/EnabledByProperty.java

```diff
@@ -43,4 +43,15 @@
    * if the specified system property exists and has the value {@code "true"}.
    */
   boolean negated() default false;
+
+  /**
+   * Default value in case the system property is not set at all.
+   */
+  String defaultValue() default "";
+
+  /**
+   * Expected property value to match for considering this bean enabled
+   */
+  String matchValue() default "true";
```

Comment:
I wonder if we need to add a flag to ignore case as well? The issue is that we used to use `Boolean.parseBoolean`, which IIRC is case insensitive. If I'm remembering that correctly, it would be good to add something like

    boolean caseSensitive default false;

That way we would be completely backwards compatible, while still allowing case-sensitive properties to be used.